cmf2_tvt120_EternalBabylon_v1.swu_public_salman_map/functions/JST/fn_respawnVehicles.sqf

```diff
@@ -307,6 +308,12 @@ JST_fnc_vehRespawn =
 	{
 		_unitVar removeMagazinesTurret [_x, [0]];
 	} forEach VehBannedMagazines;
+	// Add custom magazines
+	{
+		_unitVar addMagazineTurret _x
+	} forEach ReplaceMagazines;
+	// Save loadout for ace rearm
+	_unitVar setVariable ["ace_rearm_scriptedLoadout", true, true];

```

Comment:
Oh shit. This was supposed to be here. It must've gotten overwritten in a merge somewhere.

Reply:
No, it's definite here https://github.com/Totyano/Eternal-Babylon-TVT/blob/033ac0532ebca2794402e81141cc70094cbbafb7/cmf2_tvt120_EternalBabylon_v1.swu_public_salman_map/functions/JST/fn_respawnVehicles.sqf#L389

Reply:
Might be able to go without the one on 316 but it def needs the addMagazineTurret there

Reply:
@tanaaKa just tested without the 2nd ace_rearm and it needs to be there
The code below only gets ran at mission start, not when a vehicle respawns

Reply:
Yeah, I figured, but if we run a `setVariable` at start after the magazines change, that should save the loadout regardless.